src/middlewared/middlewared/plugins/security/update.py

```diff
@@ -73,8 +66,5 @@ async def do_update(self, data):
 
     @private
     def fips_enabled(self):
-        cp = subprocess.run(['openssl', 'list', '-providers'], capture_output=True)
-        if cp.returncode:
-            raise CallError(f'Failed to determine if fips is enabled: {cp.stderr.decode()}')
-
-        return b'OpenSSL FIPS Provider' in cp.stdout
+        # TODO: remove this method from this CRUD file
```

Comment:
It should be safe to remove this method after removing it's usages as this is a private method

Reply:
I wasn't sure if this was called anywhere in our installer code? I actually couldn't find a single occurrence of this endpoint being used anywhere?

Reply:
@yocalebo it's being used in an integration test
```
./tests/api2/test_fips.py:17:    assert call('system.security.fips_enabled') is flag
```

Reply:
fixed, thanks